protocol/contracts/beanstalk/silo/ConvertFacet.sol

```diff
@@ -75,39 +83,36 @@ contract ConvertFacet is ReentrancyGuard {
         nonReentrant
         returns (int96 toStem, uint256 fromAmount, uint256 toAmount, uint256 fromBdv, uint256 toBdv)
     {
-        address toToken; address fromToken; uint256 grownStalk; address account; bool decreaseBDV;
-
-        (toToken, fromToken, toAmount, fromAmount, account, decreaseBDV) = LibConvert.convert(convertData);
+        convertParams memory cp;
+        (cp.toToken, cp.fromToken, toAmount, fromAmount, cp.account, cp.decreaseBDV) = LibConvert.convert(convertData);
```

Comment:
This should be refactored so that cp is initialized by LibConvert.convert instead. Something like: 
```
convertParams memory cp = LibConvert.convert(convertData);
...
toAmount = cp.toAmount;
fromAmount = cp.FromAmount;

Reply:
Think i managed to do it and its fairly clean. See here [0271194](https://github.com/BeanstalkFarms/Beanstalk/pull/802/commits/027119442ab8ee76b7ea72783ec73d43a2d7751a)

Reply:
Also adjusted the naming of some variables to be consistent across these convert contracts